packages/js/src/first-time-configuration/tailwind-components/configuration-stepper-buttons.js

```diff
@@ -123,8 +130,22 @@ BackButton.defaultProps = {
  */
 export function StepButtons( { stepId, beforeContinue, continueLabel, beforeBack, backLabel } ) {
 	return <div className="yst-mt-12">
-		<ContinueButton id={ `button-${ stepId }-continue` } beforeGo={ beforeContinue }>{ continueLabel }</ContinueButton>
-		<BackButton id={ `button-${ stepId }-back` } additionalClasses="yst-ml-3" beforeGo={ beforeBack }>{ backLabel }</BackButton>
+		<ContinueButton
+			id={ `button-${ stepId }-continue` }
+			beforeGo={ beforeContinue }
+			data-hiive-event-name={ `clicked_continue | ${ HIIVE_STEPS_NAMES[ stepId ] }` }
+
+		>
+			{ continueLabel }
+		</ContinueButton>
+		<BackButton
+			id={ `button-${ stepId }-back` }
+			additionalClasses="yst-ml-3"
+			beforeGo={ beforeBack }
+			data-hiive-event-name={ `clicked_back | ${ HIIVE_STEPS_NAMES[ stepId ] }` }
+		>
+			{ backLabel }
+		</BackButton>
```

Comment:
Usage of `ContinueButton` here should be refactored to adopt the changes above, while `BackButton` should be refactored to use the same props, and then the usage here should be refactored as well

Reply:
Done!